mpl_volume_viewer.py

```diff
@@ -107,23 +107,22 @@ def process_mouse_button(self, event):
             final_dim = self.index[dim]
             point.insert(dim, final_dim)
             point = [int(round(i)) for i in point]
-            self.set_viewpoint(point)
+            self.set_viewpoint(point, dim)
```

Comment:
This is not correct when clicking: you actually want two dims when clicking (the ones *not* covered by `dim`.) ie if I click on the xy view I will need to adjust both the xz and the yz views.

I suggest instead that you add `dims=None` to `set_viewpoint`, and then use `dims=[dim]` where you know you only need one dim adjusted (as in `_dscroll`). When `dims=None` (default), it defaults to `range(3)`.

Reply:
You are right. I have modified it now.